Fix #51, #52, #53 (#55)
naruhiyo/aman: README.md

![Output](https://user-images.githubusercontent.com/16721102/101240745-8db1d100-3734-11eb-89f3-915a7f8a4a5b.gif)

### Support OS

![ubuntu](https://img.shields.io/badge/-Ubuntu-6c272d.svg?logo=Ubuntu&style=flat-square)

> ![mac](https://img.shields.io/badge/-Mac-000000.svg?logo=apple&style=flat-square)

Reviewer: - ubuntu→Ubuntu
- mac→Mac or macOS

に修正お願いします！

### Installation

#### macOS

```console
$ brew tap naruhiyo/aman
$ brew install naruhiyo/aman/aman
```

> #### Linux

Reviewer: ```suggestion
#### Ubuntu
```

に修正お願いします。

> First, download a zip binary file from github.
> 
> Next, unzip the file,

Reviewer: ```suggestion
1. Download tarball file from [GitHub releases](https://github.com/naruhiyo/aman/releases/).
2. Unpack the file.
```

```console
$ tar -zxvf xxxx.tar.gz
```

> Last, add your `$PATH`.

Reviewer: ```suggestion
3. Locate `aman` to directory to your `$PATH`.
For example, locate `/usr/local/bin`.

```console
$ mv aman_linux_amd64/aman /usr/local/bin/
\```

```

最後の`\`記号は無視してください。
ないと、suggestion表示がうまくいかなかったので。。。
